spec/src/main/resources/META-INF/smithy/bsp/extensions/jvm.smithy

```diff
@@ -115,4 +115,38 @@ list JvmOptions {
 }
 
 
+/// The build target classpath request is sent from the client to the server to
+/// query the target for its compile classpath.
+@jsonRequest("buildTarget/jvmCompileClasspath")
+operation BuildTargetClasspath {
```

Comment:
For consistency, I suggest to rename these data types to `BuildTargetJvmCompileClasspath`, `JvmCompileClasspathParams` etc.

Reply:
Makes sense, renamed